crates/wasi-nn/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "wasmtime-wasi-nn"
+version = "0.1.0"
+authors = ["The Wasmtime Project Developers"]
+description = "Wasmtime implementation of the wasi-nn API"
+documentation = "https://docs.rs/wasmtime-wasi-nn"
+license = "Apache-2.0 WITH LLVM-exception"
+categories = ["wasm", "computer-vision"]
+keywords = ["webassembly", "wasm", "neural network"]
+repository = "https://github.com/bytecodealliance/wasmtime"
+readme = "README.md"
+edition = "2018"
+
+[dependencies]
+# These dependencies are necessary for the witx-generation macros to work:
+anyhow = "1.0"
```

Comment:
Is anyhow used in the witx-generation macros, or could it be a dev-dependency?

Reply:
If I remove it:
```
error[E0433]: failed to resolve: use of undeclared type or module `anyhow`
  --> crates/wasi-nn/src/lib.rs:9:1
   |
9  | / wasmtime_wiggle::wasmtime_integration!({
10 | |     // The wiggle code to integrate with lives here:
11 | |     target: witx,
12 | |     // This must be the same witx document as used above:
...  |
25 | |     missing_memory: { witx::types::Errno::MissingMemory },
26 | | });
   | |___^ use of undeclared type or module `anyhow`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```
So it looks like it needs to be included?